tests/meta/orders/test_meta_orders_lims.py

```diff
@@ -139,7 +139,7 @@ def test_to_lims_balsamic(balsamic_order_to_submit, project):
     container_names = {sample.container_name for sample in samples if sample.container_name}
 
     # ... and pick out relevant UDFs
-    first_sample = samples[0].dict()
+    first_sample = samples[0].model_dump()
```

Comment:
A bit out of scope but why would we need this as a dict? Can't we just use first_sample.name and first_sample.udfs and so on?

Reply:
I have no idea 😅 Don't want to change the behaviour though

Reply:
Fair enough!